tests/CSharpTemp/CSharpTemp.h

```diff
@@ -700,3 +700,10 @@ class DLL_API TestVariableWithFixedArrayType
 public:
 	static Foo variableWithFixedArrayType[2];
 };
+
+class DLL_API TestOutTypeInterfaces
```

Comment:
I don't see how this test code alone causes the generation of interfaces, they are only created with multiple inheritance.